### DISCUSSION TITLE
How to clean the console after compilation is complete

### DISCUSSION THREAD

**Discussion by hcxmj** [webpack/webpack#16093] [Q&A]
When I upgraded webopack, the dev environment displayed a lot of information as shown in the following figure:
<img width="1014" alt="image" src="https://user-images.githubusercontent.com/58494807/181667699-f3752d20-84c2-45d5-8e2b-d9287bddd1f2.png">
I want to clear these console information and only show the access link, as shown in the picture below:
<img width="451" alt="image" src="https://user-images.githubusercontent.com/58494807/181668070-dba14a35-b189-40cf-9d53-0c378f68b8e2.png">
<img width="414" alt="image" src="https://user-images.githubusercontent.com/58494807/181668097-af057028-ecb0-4337-979a-39f09d24f54a.png">

I also configured to use the FriendlyErrorsWebpackPlugin plugin and devServer configuration client:{logging: 'none',}, but the console still has the above information，The query data prompts that "quiet: true" can be configured in devServer, but this attribute seems to be deleted in webpack5.x

**Comment by vankop** [comment]
`stats='errors-only'` option

**Reply by hcxmj** [reply]
ok,thank you @vankop and help me solved this problem！
